ddtrace/internal/gitmetadata.py

```diff
@@ -9,7 +9,9 @@
 from ddtrace.internal.utils import formats
 
 
-_GITMETADATA_TAGS = None  # type: typing.Optional[typing.Tuple[str, str]]
+MAIN_PACKAGE = "python_main_package"
```

Comment:
I had some doubts about it, since in `ddtrace.ext.git` all the variables are related to the git domain. But you are right, it makes more sense to have all of them defined in the same place